File: questions/q1.py
n=int(input())
sum=0
originaln=n
while n>0:
    lastdigit=n%10
    sum+=pow(lastdigit,3)
    n=n//10
    print(n)
    print(sum)

if sum==originaln:
    print("Armstrong")
else:
    print("not")

File: questions/q1.cpp
#include<iostream>
#include<cmath>
using namespace std;

int main()
{
    int n;
    cout<<"Enter a number:";
    cin>>n;

    int sum=0;
    int originaln=n;
    while(n>0){
        int lastdigit = n%10;
        sum+= (lastdigit*lastdigit*lastdigit);
        n=n/10;
        
        cout<<n<<endl;
        cout<<sum<<endl;
    }
    if(sum==originaln){
        cout<<"Armstrong";
    } else {
        cout<<"not";
    }

    return 0;
}
